### PR TITLE
NO-ISSUE: Don't raise exception when VIPs DHCP allocation is disabled.

### PR DESCRIPTION
**Pull request by eliorerz**
/cc @vrutkovs

### CODE REVIEW

File: src/assisted_test_infra/test_infra/controllers/node_controllers/nutanix_controller.py [hunk 1]
```diff
@@ -126,12 +126,13 @@ def restart_node(self, node_name: str) -> None:
...
-            if not self._entity_config.api_vip:
-                raise ValueError("API VIP is not set")
-            if not self._entity_config.ingress_vip:
-                raise ValueError("Ingress VIP is not set")
-            return {"api_vip": self._entity_config.api_vip, "ingress_vip": self._entity_config.ingress_vip}
+            if self._entity_config.api_vip and self._entity_config.ingress_vip:
+                return {"api_vip": self._entity_config.api_vip, "ingress_vip": self._entity_config.ingress_vip}
+            return self._get_random_free_vips()
 
+        return {"api_vip": "", "ingress_vip": ""}
```
Comment (vrutkovs) [inline src/assisted_test_infra/test_infra/controllers/node_controllers/nutanix_controller.py:133]: Why would we want to return values which we know are incorrect?

**Comment by vrutkovs** [comment]
/hold

Needs tests

File: src/assisted_test_infra/test_infra/controllers/node_controllers/nutanix_controller.py (hunk 1 above)
Comment (eliorerz) [reply src/assisted_test_infra/test_infra/controllers/node_controllers/nutanix_controller.py:133]: The test will fail on VIPs validations, we can raise `NotImplementedError` if we want

**Pull request closed without merging**